src/Events/Custom_Tables/V1/Schema_Builder/Abstract_Custom_Table.php

```diff
@@ -55,13 +55,28 @@ public function empty_table() {
 	public function update() {
 		$this->before_update();
 		require_once ABSPATH . 'wp-admin/includes/upgrade.php';
-		$results = (array) dbDelta( $this->get_update_sql() );
+		$query = $this->get_update_sql();
+		$this->validate_for_dbDelta($query);
+		$results = (array) dbDelta( $query );
 		$this->sync_stored_version();
 		$results = $this->after_update( $results );
 
 		return $results;
 	}
 
+	/**
+	 * Inspects query strings being passed to dbDelta, and logs an error if not ideal.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $query Query string to inspect for case sensitivity before using in dbDelta
+	 */
+	public function validate_for_dbDelta( string $query ) {
```

Comment:
Avoid the camelcase method.

Reply:
The function it referenced is camel cased.